### PR TITLE
Spsh 329 user general overview mobile

### PR DESCRIPTION
**Pull request by godtierbatuhan**
# Description

- Fixed responsiveness for the user details vue.
- Added hover animation on small screens.
- Added on hover color change.
- Hid the "Close" button for the LayoutCard.vue on small screens.
- Fixed margin space according to UX design for all screens.
- Fixed footer logos width on xs screen.
- Added button animation on max width 600px to SASS variables.

## Approval for review

- [ ] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: src/components/layout/TheFooter.vue [hunk 1]
```diff
@@ -236,7 +236,22 @@
 
   @media (max-width: 600px) {
     .sponsor-logo {
+      width: 100%;
       width: 160px; /* Set the width to 100px or less */
```
Comment (jaggarnaut) [inline src/components/layout/TheFooter.vue:240]: width ist doppelt vergeben

File: src/components/cards/LayoutCard.vue [hunk 2]
```diff
@@ -11,9 +11,10 @@
   <v-card class="layout-card">
     <v-row
       align="center"
-      class="ml-5 mt-0"
+      class="ml-md-6 ml-1 my-1"
     >
-      <v-col>
+  
+      <v-col cols="auto">
         <h2 class="text-left layout-card-headline">{{ header }}</h2>
```
Comment (jaggarnaut) [inline src/components/cards/LayoutCard.vue:18]:
```suggestion
        <h2 class="text-left headline-2">{{ header }}</h2>
```
Altlast, aber mit der Klasse ändert sich die Schriftgröße responsiv

Comment (jaggarnaut) [reply src/components/cards/LayoutCard.vue:18]: Bitte auch in main.scss die Zeilen 33-36 löschen.

File: src/views/admin/UserDetailsView.vue [hunk 3]
```diff
@@ -75,26 +84,38 @@
...
+          <v-col>
+            <h3 class="subtitle-1">{{ $t('user.password') }}</h3>
+          </v-col></v-row
+        >
+        <v-row
+          justify="end"
+          class="mr-lg-10"
+        >
+          <v-col
+            v-col
```
Comment (jaggarnaut) [inline src/views/admin/UserDetailsView.vue:102]:
```suggestion
```
v-col ist doppelt

Comment (godtierbatuhan) [reply src/views/admin/UserDetailsView.vue]: Ah, ist nach dem backmerge wieder da *.*

**Review by jaggarnaut — APPROVED** [review]

**Pull request merged**